hca/define_api.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+# import os, sys, json, time, base64
+import json, sys, os
+from io import open
+# import pprint
+import requests
+from .parser import get_parser
+
+
+class Api:
+    """Class for interacting with the REST Api."""
```

Comment:
The parser building all takes place in parser.py. I was intending this class to actually make the requests given the cmd line input.